### PR TITLE
Update language css class -- Fixes #15

### LINKED ISSUE #15: Inline language declaration throws unknown language
I'm using this with the markdown and html loaders as shown in the examples here. However, if my markdown code block uses inline language declaration then it throws errors about an unknown language.

```
ERROR in ./src/components/switch/docs/readme.md
Module build failed: Error: Unknown language: "language-typescript"
    at highlight (C:\dev\WebCommon-HtmlShare\node_modules\highlight.js\lib\highlight.js:494:13)
    at highlightCode (C:\dev\WebCommon-HtmlShare\node_modules\highlight-loader\index.js:50:12)
    at Object.<anonymous> (C:\dev\WebCommon-HtmlShare\node_modules\highlight-loader\index.js:38:12)
    at initialize.<anonymous> (C:\dev\WebCommon-HtmlShare\node_modules\cheerio\lib\api\manipulation.js:275:74)
    at exports.domEach (C:\dev\WebCommon-HtmlShare\node_modules\cheerio\lib\utils.js:52:24)
    at initialize.exports.replaceWith (C:\dev\WebCommon-HtmlShare\node_modules\cheerio\lib\api\manipulation.js:268:3)
    at Object.module.exports (C:\dev\WebCommon-HtmlShare\node_modules\highlight-loader\index.js:25:13)
 @ ./src/components/switch/docs/index.ts 4:0-48 4:0-48
 @ ./src/docs.ts
 @ multi (webpack)-dev-server/client?http://localhost:8080 ./src/docs.ts
i ｢wdm｣: Failed to compile.
```

the portion causing the problem in the md file:
```
```typescript
import { Switch } from '@salespad/html-share';
const switch = new Switch(document.getElementById('switch-example'));
/```
```

and the relevant portion of the config:
```
{
          test: /\.md$/,
          loader: 'html-loader!highlight-loader!markdown-loader'
}
```

### PR DESCRIPTION
**Pull request by aronduby**
markdown is outputting code tag with classes of language-[name of language], updated this to match

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ module.exports = function(input) {
 
     var text = $e.text();
     var klass = $e.attr('class') || '';
-    var lang = klass.split('lang-').filter(id);
+    var lang = klass.split('language-').filter(id);
```
Comment (bebraw) [inline index.js]: I think this should support both options so we don't break backwards compatibility.

Comment (aronduby) [reply index.js]: good call, update submitted

**Comment by bebraw** [comment]
Published. Thanks.

**Comment by aronduby** [comment]
thank you for getting it in and published right away

**Pull request merged**